presto-client/src/main/java/io/prestosql/client/SpnegoHandler.java

```diff
@@ -100,7 +102,16 @@ public SpnegoHandler(
         this.keytab = requireNonNull(keytab, "keytab is null");
         this.credentialCache = requireNonNull(credentialCache, "credentialCache is null");
 
-        kerberosConfig.ifPresent(file -> System.setProperty("java.security.krb5.conf", file.getAbsolutePath()));
+        kerberosConfig.ifPresent(file -> {
+            String newValue = file.getAbsolutePath();
+            String currentValue = System.getProperty("java.security.krb5.conf");
+            checkState(
+                    currentValue == null || Objects.equals(currentValue, newValue),
+                    "Refusing to set system property 'java.security.krb5.conf' to '%s', it is already set to '%s'",
+                    newValue,
+                    currentValue);
+            System.setProperty("java.security.krb5.conf", newValue);
```

Comment:
Can you extract an utility for that in presto-plugin-toolkit? And reuse it in other places.

Reply:
i thought about this.
presto-client doesn't (and shouldn't) depend on presto-plugin-toolkit
and i didn't want to introduce another module for these few lines